src/keyboard.tsx

```diff
@@ -0,0 +1,33 @@
+import { Component, ReactNode } from 'react';
+
+type State = {
+  key: string,
+};
+
+export class Keyboard extends Component<{}, State> {
+  state: Readonly<State> = {
+    key: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.keySaver);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.keySaver);
+  }
+
+  keySaver = (event: KeyboardEvent) => {
```

Comment:
function name should be a verb, not a noun
it will be better to name it like an action that it does
'saveKey' or 'handlePressedKey' and so on